decompiler/IR2/AtomicOpTypeAnalysis.cpp

```diff
@@ -662,7 +662,7 @@ TP_Type SimpleExpression::get_type_int2(const TypeState& input,
     }
   }
 
-  if (env.version == GameVersion::Jak2 && tc(dts, TypeSpec("symbol"), arg1_type) &&
+  if (env.version >= GameVersion::Jak2 && tc(dts, TypeSpec("symbol"), arg1_type) &&
```

Comment:
I think this one should be reverted. String->symbol is different and shouldn't use the jak2's constant.